.yarnrc.yml

```diff
@@ -0,0 +1,13 @@
+enableScripts: false
+
+logFilters:
```

Comment:
Nit: we should disable telemetry here as well, as was done in the module template

Reply:
done [here](https://github.com/MetaMask/eth-simple-keyring/pull/119/commits/71ab402fa900f3ee90445b5f6a072d69aed7caa2)